build/Build.cs

```diff
@@ -11,8 +11,8 @@ partial class Build : NukeBuild
     [Parameter("Configuration to build. Default is 'Release'")]
     readonly Configuration BuildConfiguration = Configuration.Release;
 
-    [Parameter("Platform to build - x86 or x64. Default is 'x64'")]
-    readonly MSBuildTargetPlatform Platform = MSBuildTargetPlatform.x64;
+    [Parameter("Platform to build - x86, x64, ARM64. Default is current architecture.")]
```

Comment:
nit: Provide a better clarity her - "Default is current architecture."

Reply:
any suggestions?